brutils/cep.py

```diff
@@ -1,4 +1,11 @@
+from json import loads
 from random import randint
+from unicodedata import normalize
+from urllib.request import urlopen
+
+from .data.enums import UF
+from .exceptions import CEPNotFound, InvalidCEP
+from .types import Address
```

Comment:
Como diz na [Pep 8](https://peps.python.org/pep-0008/#imports), o ideal seria deixar esses imports explícitos, devido a estrutura do projeto não ser tão complexa.
 
"Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):
import mypkg.sibling
from mypkg import sibling
from mypkg.sibling import example
However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:

from . import sibling
from .sibling import example
Standard library code should avoid complex package layouts and always use absolute imports."